src/components/VolunteerTable/VolunteerTable.tsx

```diff
@@ -0,0 +1,187 @@
+  name: string,
+  type: number,
+  email: string,
+  location: string
+) {
+  return { name, type, email, location };
+}
+
+function getUserRole(userType: number): string {
+  switch (userType) {
+    case 0:
+      return "Corporate Team";
+    case 1:
+      return "Community Group";
+    case 2:
+      return "Individual Volunteer";
+    default:
+      return "undefined user role";
+  }
+}
+
+const rows = [
+  createData("Name1", 0, "email1", "location1"),
+  createData("Name2", 1, "email2", "location2"),
+  createData("Name3", 2, "email3", "location2"),
+];
+
+export default function VolunteerTable() {
+  return (
+    <TableContainer
+      sx={{
+        border: "solid 1px #E4E7EC",
+        borderRadius: "12px",
+        boxShadow: "0px 2px 4px -2px #1018280F, 0px 4px 8px -2px #1018281A",
+      }}
+    >
+      <Table
+        sx={{
+          minWidth: 650,
+          borderColor: "#E4E7EC",
+        }}
+        aria-label="simple table"
+      >
+        <TableHead>
+          <TableRow sx={{ backgroundColor: "#F9FAFB" }}>
+            <TableCell
+              sx={{
+                color: "#667085",
+                fontWeight: "bold",
+                height: "44px",
+                width: "255px",
+              }}
+            >
+              Name
+            </TableCell>
+            <TableCell
+              align="left"
+              sx={{
+                color: "#667085",
+                fontWeight: "bold",
+                height: "44px",
+                width: "255px",
+              }}
+            >
+              Type
+            </TableCell>
+            <TableCell
+              align="left"
+              sx={{
+                color: "#667085",
+                fontWeight: "bold",
+                height: "44px",
+                width: "255px",
+              }}
+            >
+              Email address
+            </TableCell>
+            <TableCell
+              align="left"
+              sx={{
+                color: "#667085",
+                fontWeight: "bold",
+                height: "44px",
+                width: "255px",
+              }}
+            >
+              Location
+            </TableCell>
+            <TableCell
+              sx={{ width: "116px", height: "44px  " }}
+              align="center"
+            ></TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody sx={{ borderColor: "pink" }}>
+          {rows.map((row) => (
+            <TableRow
+              key={row.name}
+              sx={{
+                "&:last-child td, &:last-child th": { border: 0 },
+                borderColor: "pink",
+              }}
+            >
+              <TableCell
+                sx={{
+                  borderColor: "#E4E7EC",
+                  fontWeight: "bold",
+                  color: "#101828",
+                }}
+                component="th"
+                scope="row"
+              >
+                <div className="flex items-center gap-4">
```

Comment:
Small thing: I think you guys forgot to push but to fix that small issue with the flex, I just moved the flex into a separate div inside the TableCell